examples/nextjs-spa/styles/Home.module.css

```diff
@@ -0,0 +1,129 @@
+.container {
```

Comment:
Do we use this css file anywhere?

Reply:
Removed, was a default Next css file. They use Home instead of Dashboard, maybe we can rename Dashboard.